docs/sphinx/_toc.yml.in

```diff
@@ -47,6 +47,7 @@ subtrees:
           - file: how-to/hip_runtime_api/memory_management/stream_ordered_allocator
       - file: how-to/hip_runtime_api/cooperative_groups
       - file: how-to/hip_runtime_api/hipgraph
+      - file: how-to/hip_runtime_api/call_stack
```

Comment:
Please also add it to the index to the appropriate section

Reply:
Thanks! Added.